webpack.prod.js

```diff
@@ -1,11 +1,13 @@
 'use strict';
 const merge = require('webpack-merge');
-const common = require('./webpack.common.js');
-const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const styled = require('./webpack.styled.js');
+const unstyled = require('./webpack.unstyled.js');
 
-module.exports = merge(common, {
-	mode: 'production',
-	plugins: [
-		new CleanWebpackPlugin()
```

Comment:
The CleanWebpackPlugin needed to go, because with it, it would only keep the files of one of the two builds.

Reply:
How is it possible that it does work with all NLO verticals? 🤔 

https://github.com/nederlandseloterij/NLO.Lotto/blob/develop/src/NLO.Lotto/webpack.prod.js

Reply:
Haha I don't have access to this organisation anymore °~°